### PR TITLE
pybind/tox: pass additional command line arguments through to tox

### PR DESCRIPTION
**Pull request by smithfarm**
Fixes: 9426f1f2045d0ae0f319530c3dc3a9240d838d07
Fixes: https://tracker.ceph.com/issues/39579
Signed-off-by: Nathan Cutler <ncutler@suse.com>

### CODE REVIEW

**Comment by smithfarm** [comment]
Note to self: once this merges, cherry-pick it into #27884

**Comment by LenzGr** [comment]
jenkins test docs

File: src/pybind/mgr/orchestrator_cli/run-tox.sh [hunk 1]
```diff
@@ -40,7 +40,7 @@ ENV_LIST=$(echo "$ENV_LIST" | sed -e 's/,$//')
 # use bash string manipulation to strip off any trailing comma
 ENV_LIST=${ENV_LIST%,}
 
-tox -c "${TOX_PATH}" -e "${ENV_LIST}"
+tox -c "${TOX_PATH}" -e "${ENV_LIST}" "@"
```
Comment (tchaikov) [inline src/pybind/mgr/orchestrator_cli/run-tox.sh]: s/@/$@/

Comment (smithfarm) [reply src/pybind/mgr/orchestrator_cli/run-tox.sh]: ouch! thanks for catching that, @tchaikov

**Comment by smithfarm** [comment]
I guess this doesn't need a teuthology run. Passing make check should be enough testing.

**Pull request merged**